### ISSUE TITLE
Configurable 'admin' email address

### ISSUE THREAD

**Issue by stuartlewis** [DataVault/datavault#137]
Configurable 'admin' email address for display and contact emails when things go wrong.

**Comment by robintaylor** [comment]
Hi,

This begs the question, where do we put it? We don't currently have a run time config file/directory, or a config table in the database.

**Comment by stuartlewis** [comment]
Yep - it hinges on https://github.com/DataVault/datavault/issues/24

**Comment by tomhigginsuom** [comment]
I've been putting the file store config in the DB - my line of thinking was that if the information needed to be updated at run-time (e.g. to add config for a new user, to change existing config values) then the DB would be a good place for it.

Perhaps we need a general config table for some of these one-off values?
